tools/rex/Makefile

```diff
@@ -0,0 +1,43 @@
+SHELL        := /usr/bin/env bash
+SRC          := Main.hs
+VERSION      := $(shell sed -n 's/^version: *\(.*\)$$/\1/p' $(CURDIR)/rex.cabal)
+BUILD_NUMBER ?= 0
+DOCKER       ?= true
+ACI          := rex-$(VERSION)b$(BUILD_NUMBER)_linux_amd64.aci
+
+include ../../mk/publish.mk
+
+
+default: dist
+
+.PHONY: dist
+dist: clean $(ACI)
+
+.PHONY: build
+build: rex
+
+.PHONY: publish
+publish: $(ACI) $(ACI).asc s3-exists/$(ACI) s3-upload/$(ACI)
```

Comment:
some of those targets don't exist anymore?

Reply:
Yeah, and a few more things need fixing.
- there's an include that doesn't work anymore
- we should probably stack install into `dist` as with the other tools

I'm not quite sure how rex is supposed to be packaged/used, so maybe someone else could take care of these outstanding issues with the Makefile?